ja-JP-mac/dom/chrome/accessibility/mac/accessible.properties

```diff
@@ -1,73 +1,73 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-jump	=移動
-press	=押す
-check	=チェックを入れる
-uncheck	=チェックを外す
-select	=選択
-open	=開く
-close	=閉じる
-switch	=切り替え
-click	=クリック
-collapse=折りたたむ
-expand	=展開する
-activate=動作させる
-cycle	=切り替え
-# An action provided to accessibility clients such as screen readers to allow
-# them to click an element when the click will be handled by a container
-# (ancestor) element. This is not normally reported to users.
-# (^m^) Bug 1395181 で追加。Entity 名にスペースが入っているためコメントアウト
-# https://phabricator.services.mozilla.com/D144252#inline-797003
-#click ancestor = 祖先要素をクリック
-
-# Universal Access API support
-# (Mac Only)
-# The Role Description for AXWebArea (the web widget). Like in Safari.
-htmlContent	=HTML コンテント
-# The Role Description for the Tab button.
-tab		=タブ
-# The Role Description for definition list dl, dt and dd
-term		=用語
-definition	=定義
-# The Role Description for an input type="search" text field
-searchTextField	=テキスト検索フィールド
-# Role Description (exposed as AXTitle) for datepickers
-dateField	=日付フィールド
-# The Role Description for WAI-ARIA Landmarks
-application	=アプリケーション
-search		=検索
-banner		=バナー
-navigation	=ナビゲーション
-complementary	=補完コンテント
-content		=コンテント
-main		=メイン
-# The (spoken) role description for various WAI-ARIA roles
-alert		=警告
-alertDialog	=警告ダイアログ
-dialog		=ダイアログ
-article		=記事
-document	=文書
-# The (spoken) role description for the WAI-ARIA figure role
-# https://w3c.github.io/aria/core-aam/core-aam.html#role-map-figure
-figure		=図
-# The (spoken) role description for the WAI-ARIA heading role
-# https://w3c.github.io/aria/core-aam/core-aam.html#role-map-heading
-heading		=見出し
-log		=ログ
-marquee		=マーキー
-math		=数式
-note		=注釈
-region		=領域
-status		=アプリケーション状態
-timer		=タイマー
-tooltip		=ツールチップ
-separator	=区切り
-tabPanel	=タブパネル
-# The roleDescription for the html:mark element
-highlight	=強調
-# The roleDescription for the details element
-details		=詳細
-# The roleDescription for the summary element
-summary		=要約
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+jump	=移動

```

Comment:
Looks like you change line ending? We should probably stick to LF, not CRLF.

In case, let me know and I can help convert them.

Reply:
Sorry, they are changed by my lot-conv.py script. I will fix the bug.

Reply:
Fixed.
https://github.com/marsf/lot-conv/commit/992d9e69c2fbb3acbff913880e66a77bb57a671e